great-idea/css/index.css

```diff
@@ -63,4 +63,118 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/*------------------------------------------------------------*/
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	background: ;
+}
+
+p {
+	line-height: 1.2;
+}
+.container {
+	width: 876px;
+	padding: 30px;
+	box-sizing: border-box;
```

Comment:
You don't actually need to specify `box-sizing: border-box` on this element, as you've already applied that to every element with the universal selector